src/app/search/advancedsearch/advancedsearch.component.html

```diff
@@ -13,116 +13,99 @@
   ~    See the License for the specific language governing permissions and
   ~    limitations under the License.
   -->
-<div
-  *ngIf="isModalShown$ | async"
-  [config]="{ show: true }"
-  class="modal fade"
-  bsModal
-  #advancedSearchModal="bs-modal"
-  tabindex="-1"
-  role="dialog"
-  aria-labelledby="exampleModalLongTitle"
-  aria-hidden="true"
-  (onHidden)="onHidden()"
->
-  <div class="modal-dialog">
-    <div class="modal-content">
-      <div class="modal-header">
-        <h4 class="modal-title pull-left">Advanced Search</h4>
-        <button type="button" class="close pull-right" aria-label="Close" (click)="advancedSearchModal.hide()">
-          <span aria-hidden="true">&times;</span>
-        </button>
+<h4 mat-dialog-title class="pull-left">Advanced Search</h4>
+<mat-dialog-content class="h-100 p-0 m-0">
+  <div mat-dialog-actions class="modal-title p-0 m-0 dialog-actions">
+    <button type="button" class="close pull-right" aria-label="Close" tabindex="-1">
+      <span aria-hidden="true">&times;</span>
+    </button>
+  </div>
+</mat-dialog-content>
+<div mat-dialog-content>
+  <div mat-dialog-actions class="modal-body form-horizontal dialog-actions p-0 mb-0">
```

Comment:
might as well ditch as many modal bootstrap classes as possible: modal-body, modal-title, etc.  i think it's interfering with the material directives, which is probably why you have a lot of padding and margin changes.

Reply:
Removed the modal bootstrap classes as well as most padding and margin changes.